CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.1.0 (2020-06-15)
+
+- Added `ariaLabel` and `ariaLabelledBy` input properties to the tabset component. [#90](https://github.com/blackbaud/skyux-tabs/pull/90)
+- Fixed `aria-controls` and `role` HTML properties on the tab component to follow proper accessibility guidelines. [#90](https://github.com/blackbaud/skyux-tabs/pull/90)
```

Comment:
Should `tab component` be `tabset component`? In the PR we link to, I don't see any updates to `tab.component.ts`, but I do see updates to `tabset.component.ts`.